src/components/datasets/DatasetForm.js

```diff
@@ -1,56 +1,43 @@
 import PropTypes from "prop-types";
 
 import { Form, Input } from "antd";
+
 const { Item } = Form;
 
 import DataUseInput from "../DataUseInput";
 
 import { DATA_USE_PROP_TYPE_SHAPE, INITIAL_DATA_USE_VALUE } from "@/duo";
+import { useDatsValidator } from "@/hooks";
+import { useDiscoveryValidator } from "@/modules/metadata/hooks";
 import { simpleDeepCopy } from "@/utils/misc";
+import DropBoxJsonSelect from "../manager/dropBox/DropBoxJsonSelect";
 
-const validateJson = (rule, value) => {
-  try {
-    JSON.parse(value);
-    return Promise.resolve();
-  } catch (e) {
-    return Promise.reject("Please enter valid JSON");
+const DatasetForm = ({ initialValue, form }) => {
+  const discoveryValidator = useDiscoveryValidator();
+  const datsValidator = useDatsValidator();
+  if (initialValue && !initialValue?.data_use) {
+    initialValue["data_use"] = simpleDeepCopy(INITIAL_DATA_USE_VALUE);
```

Comment:
we can't mutate object props, since it violates the react data flow (and might have weird effects up the tree) - i think the true used initial value should be memoized, especially with a deep copy present.

Reply:
done